core/node/eth_sender/src/aggregator.rs

```diff
@@ -320,30 +320,42 @@ impl Aggregator {
             return None;
         }
 
-        if let Some(version_id) = storage
+        let minor_version = storage
             .blocks_dal()
             .get_batch_protocol_version_id(batch_to_prove)
             .await
             .unwrap()
-        {
-            let verifier_config_for_next_batch = storage
-                .protocol_versions_dal()
-                .l1_verifier_config_for_version(version_id)
-                .await
-                .unwrap();
-            if verifier_config_for_next_batch != l1_verifier_config {
-                return None;
-            }
-        }
-        let proofs =
-            load_wrapped_fri_proofs_for_range(batch_to_prove, batch_to_prove, blob_store).await;
+            .unwrap();
+        let allowed_patch_versions = storage
```

Comment:
please add comment explaining what we're doing here (that there might be multiple verification keys, and that l1_verifier_config tells us what VK L1 can accept)